vocabularies/earth-science-data-category.ttl

```diff
@@ -558,3 +557,11 @@
     skos:notation "040308" ;
     skos:prefLabel "Palaeontology"@en .
 
+:lodgement-form a skos:Collection ;

```

Comment:
If we reduce the complexity of the ESDC for entry where will those vocabs be used?  How we will define data as a certain type? i.e. if a airborne survey is classified as geophysics on lodgement.  Where does it get defined as an airborne EM survey with a ESDC of electromagnetic methods in geophysics being more precise?   Similarly for seismic data.  How does it get tagged as seismic data from this?